app/forms/idv/api_document_verification_form.rb

```diff
@@ -106,10 +106,25 @@ def throttle
     end
 
     def validate_image_urls
-      errors.add(:front_image_url, invalid_link) unless valid_url?(:front_image_url)
-      errors.add(:back_image_url, invalid_link) unless valid_url?(:back_image_url)
+      unless valid_url?(:front_image_url)
+        errors.add(
+          :front_image_url, invalid_link,
+          type: :doc_verification
```

Comment:
so IIRC from the ticket, the purpose of the `type:` was to identity which error message, so I think this should be `:invalid_link`
```suggestion
          type: :invalid_link,
```